app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -1042,13 +1042,13 @@ class PlayerFragment : Fragment(), OnlinePlayerOptions {
     /**
      * Can be used for autoplay and manually skipping to the next video.
      */
-    private fun playNextVideo(nextId: String? = null) {
+    private fun playNextVideo(nextId: String? = null, isOnQueueClick: Boolean = false) {
         if (nextId == null && PlayingQueue.repeatMode == Player.REPEAT_MODE_ONE) {
             viewModel.player.seekTo(0)
             return
         }
 
-        if (!PlayerHelper.isAutoPlayEnabled(playlistId != null)) return
+        if (!PlayerHelper.isAutoPlayEnabled(playlistId != null) && !isOnQueueClick) return
```

Comment:
Couldn't we instead do `!PlayerHelper.isAutoPlayEnabled(playlistId != null) && nextId == null`, since in theory autoplay will never call this function with any video id provided (if I remember correctly how I intended to write that code) whereas the nextId is set when tapping a video in the queue?

Reply:
(same applies for your other PR, I probably merged that too quickly before thinking)

Reply:
yep, it worked in that way too, I opened another PR